src/main/resources/db/migration/V2_0__add_fields_from_sykmelding.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE BEHANDLER_KONTOR ADD COLUMN system VARCHAR(100);
+ALTER TABLE BEHANDLER ADD COLUMN KATEGORI CHAR(2) NOT NULL DEFAULT 'LE';
```

Comment:
Er `LE` Fastlege?

Reply:
Typen vi hadde fra før er flyttet til BEHANDLER_ARBEIDSTAKER. Den er FASTLEGE for de som finnes nå, men vil etterhvert også kunne være SYKMELDER. Så en og samme lege kan være FASTLEGE for en arbeidstaker, men SYKMELDER for en annen.

Denne nye typen på behandler vil være et subset av helsepersonellkategori. De vi har nå er leger (LE), men her vil det kunne komme andre koder for andre behandlere (feks TL, FT og KI).

Reply:
Skjønner. Hvor kommer dette "kodeverket" fra, kuhr? Ikke så tilhenger slikt koder, så om det er mulig å legge til eller "mappe" over til noe er lettere å forholde seg til så hadde det vært fint.
Typ: FYSIOTERAPEUT, KIROPRAKTOR, LEGE,...,.

Reply:
Ja, det kan vi godt gjøre 👍